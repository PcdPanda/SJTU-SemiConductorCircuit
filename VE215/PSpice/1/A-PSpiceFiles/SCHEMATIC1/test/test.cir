** Profile: "SCHEMATIC1-test"  [ D:\PANDA\Study\VE215\PSpice\1\A-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 0.0005 0 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
