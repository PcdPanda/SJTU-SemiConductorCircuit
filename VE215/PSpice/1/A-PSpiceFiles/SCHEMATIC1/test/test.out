
**** 11/02/17 00:35:20 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\PANDA\Study\VE215\PSpice\1\A-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.TRAN  0 0.0005 0 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source A
V_V1         N01221 0  
+PULSE -0.5 0.5 0 0 0 0.05m 0.1m
R_R1         N02044 N001110  100 TC=0,0 
C_C1         0 N00682  820p IC=0 TC=0,0 
L_L1         N01221 N02044  1m  
R_R2         N001110 N00682  7500 TC=0,0 

**** RESUMING test.cir ****
.END

**** 11/02/17 00:35:20 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\PANDA\Study\VE215\PSpice\1\A-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(N00682)-131.6E-09 (N01221)    -.5000 (N02044)    -.5000 (N001110)    -.4934    




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1         6.579E-05

    TOTAL POWER DISSIPATION   3.29E-05  WATTS



          JOB CONCLUDED

**** 11/02/17 00:35:20 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ D:\PANDA\Study\VE215\PSpice\1\A-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =        0.00
